convert.go

```diff
@@ -87,6 +87,8 @@ func Convert(ctx context.Context, img string, opts ...ConvertOption) error {
 	format := strings.TrimPrefix(filepath.Ext(o.output), ".")
 	if format == "" {
 		format = "raw"
+	} else if format == "vhd" {
+		format = "vpc"
```

Comment:
Can you move that in builder.go where the validation is implemented ?
Here, where we only ensure some defaults.